tests/test_linters.py

```diff
@@ -341,15 +342,23 @@ def test_framework_operator_no_ops_imported(tmp_path, monkeypatch, import_line):
 @pytest.mark.parametrize(
     "charm_name, charm_module", [("foobar", "foobar.py"), ("foo-bar", "foo_bar.py")]
 )
-def test_framework_reactive_used_ok(tmp_path, import_line, charm_name, charm_module):
+def test_framework_reactive_used_ok(create_config, import_line, charm_name, charm_module):
     """The reactive framework was used.
 
     Parametrized args:
     - import_line: different ways to express the import
     """
-    # metadata file with needed name field
-    metadata_file = tmp_path / "metadata.yaml"
-    metadata_file.write_text(f"name: {charm_name}")
+    tmp_path = create_config(
```

Comment:
This is counterintuitive. Pytest will use the same `tmp_path` for all fixtures passed to a single test run, so it would be better to keep `tmp_path` as a direct fixture used by this test than to have `create_config` return the path.

Reply:
OK